python/setup.cfg

```diff
@@ -0,0 +1,62 @@
+#   LICENSE
+# Copyright (c) 2021 Genome Research Ltd.
+# Author: Cancer Genome Project cgphelp@sanger.ac.uk
+#
+#
+# This file is part of CASM-Smart-Phase.
+#
```

Comment:
This is inconsistent with `python/run_tests.sh`.  The blank lines seem to be all over the place in the licenses